tests/crypto/test_ccm_mode/testcase.yaml

```diff
@@ -0,0 +1,3 @@
+tests:
+-   test:
+        tags: crypto aes ccm
```

Comment:
Moving things around and copying things to a different path makes this very difficult to manage and all of the sudden a test case that appears in a report as XYZ is not ABC because the path changed. I do not see this as duplication, the "database" is the file, not its location in the git tree.